addons/resource.language.en_gb/resources/strings.po

```diff
@@ -7185,7 +7185,30 @@ msgctxt "#13461"
 msgid "Enable this option to use hardware acceleration for the HEVC codec. If disabled the CPU will be used instead."
 msgstr ""
 
-#empty strings from id 13462 to 13504
+#: system/settings/settings.xml
+msgctxt "#13462"
```

Comment:
contextual comment is missing. Something like: `#.label of a setting to enable the PRIME render engine` (or whatever it actually is)